CHANGES

```diff
@@ -1,5 +1,26 @@
 This file describes the most significant changes. For more detail, use
 'git log' on a clone of the charm repository.
+================================================================================
+What's new in Charm++ 7.1.0
+================================================================================
+
+This is a feature release, with the following major changes:
+
+Highlights:
+
+- The License has changed to Apache 2.0 with LLVM exception.  This
+  change to a popular Open Source license is intended to simplify use,
+  collaboration, and greater community involvement in the development
+  of Charm++.
+
+- The CkIO library has been enhanced to support both input and output.
+ <insert more details here>
+
+- The OFI layer has been extended to support the CXI (Cassini)
```

Comment:
I would add that explicitly adding cxi to the build command is not necessary on Cray systems. Also, I would hope that we print something saying that cxi is enabled at startup when it is on.

Reply:
Note added.  PR https://github.com/UIUC-PPL/charm/pull/3810 adds output at startup when CXI is in use.